configs/droid.kmap

```diff
@@ -1,5 +1,4 @@
 keycode 100 = AltGr
-keycode 102 = Home
```

Comment:
Why is this removed? Affecting home button on external keyboards?

Reply:
Yes. Apparently there has been complaints in TJC that we broke terminal home key so better remove this one.

Reply:
Only known place where we use Qt.Key_Home was removed, but kept it for now to be on the safe side.